src/app/app.component.html

```diff
@@ -19,8 +19,17 @@ <h1>Without preview</h1>
                [imagePreviewTemplate]="singleNav"
                [bottomViewerTemplate]="bottomViewerTemplate"></tsq-gallery>
 
+  <h1>Display viewer inline</h1>
+  <div> <input type="checkbox" [checked]="hasMiniPreview" (change)="hasMiniPreview = !hasMiniPreview"> Show minis</div>
+  <div style="height: 600px;">
```

Comment:
Even though this is only the demo page, I think it's interesting to pass this to a css class.